cypress/e2e/top-navigation.cy.ts

```diff
@@ -0,0 +1,112 @@
+/// <reference types='cypress' />
+
+describe("Top navigation test", () => {
+  beforeEach(() => {
+    cy.visit("/");
+    cy.get('.navbar a[href="/"]').as("home");
+    cy.get('.navbar a[href="/projects"]').as("projects");
+    cy.get('.navbar a[href="/talks"]').as("talks");
+    cy.get('.navbar a[href="/blog"]').as("blog");
+    cy.get('.navbar a[href="#hire-me"]').as("contact");
+  });
+  it("Should check if top navigation buttons have correct addresses", () => {
+    cy.get("@home")
+      .invoke("text")
+      .then((text1) => {
+        cy.get(".navbar")
+          .contains("Home")
+          .invoke("text")
+          .then((text2) => {
+            expect(text1).to.equal(text2);
+          });
+      });
+    cy.get("@projects")
+      .invoke("text")
+      .then((text1) => {
+        cy.get(".navbar")
+          .contains("Projects")
+          .invoke("text")
+          .then((text2) => {
+            expect(text1).to.equal(text2);
+          });
+      });
+    cy.get("@talks")
+      .invoke("text")
+      .then((text1) => {
+        cy.get(".navbar")
+          .contains("Talks")
+          .invoke("text")
+          .then((text2) => {
+            expect(text1).to.equal(text2);
+          });
+      });
+    cy.get("@blog")
+      .invoke("text")
+      .then((text1) => {
+        cy.get(".navbar")
+          .contains("Blog")
+          .invoke("text")
+          .then((text2) => {
+            expect(text1).to.equal(text2);
+          });
+      });
+    cy.get("@contact")
+      .invoke("text")
+      .then((text1) => {
+        cy.get(".navbar")
+          .contains("Contact")
+          .invoke("text")
+          .then((text2) => {
+            expect(text1).to.equal(text2);
+          });
+      });
+  });
+
+  it("Should mimic real user interactions with top navigation", () => {
+    function goBackToHome(cy) {
```

Comment:
```suggestion
    function goBackToHome() {
```

you do not need this argument, because `cy` is a global